R/fct_SQLite.R

```diff
@@ -369,3 +496,72 @@ db_get_review <- function(
       dplyr::as_tibble()
   })
 }
+
+update_db_version <- function(db_path, version = "1.1") {
+  stopifnot(file.exists(db_path))
+  version <- match.arg(version)
+  temp_path <- withr::local_tempfile(fileext = ".sqlite")
+  file.copy(db_path, temp_path)
+  con <- get_db_connection(temp_path)
+  
+  current_version <- tryCatch({
+    DBI::dbGetQuery(con, "SELECT version FROM db_version") |> 
+      unlist(use.names = FALSE)}, error = \(e){""})
+  if(identical(current_version, db_version)) return("Database up to date. No update needed")
```

Comment:
It probably should only update if the version is currently missing, or if the version is at least smaller than 1.1, right? Just to protect us in the future

Reply:
If we need this type of check in the future, we can add it then.